### PR TITLE
#189 separate review instructions for log4net

### PR DESCRIPTION
**Pull request by FreeAndNil**
see https://logging.apache.org/log4net/release/release-review-instructions.html

### CODE REVIEW

**Review by fluffynuts — APPROVED** [review]

File: src/site/xdoc/release/release-review-instructions.xml
Comment (vy) [inline src/site/xdoc/release/release-review-instructions.xml:1]: Given Log4Net primarily targets Windows users, I think we should cater for them in the instructions. For instance, `umask`, `wget`, `cd $_`, `sha512sum`, `for sigFile ...`, etc. don't work in Windows. I suggest

- Separating Windows- and Linux-specific instructions
- Rewriting Windows-specific instructions

**An alternative route** can be to start with a warning telling Windows users to install bash/cygwin/subsystem/etc. to be able to run following Linux commands.

**Review by vy — CHANGES_REQUESTED** [review]

File: src/site/xdoc/release/release-review-instructions.xml
Comment (FreeAndNil) [reply src/site/xdoc/release/release-review-instructions.xml:1]: @vy thanks for the hint.
I've added a PowerShell-Script for Windows users and updated the site accordingly.

File: doc/MailTemplate.txt
Comment (vy) [inline doc/MailTemplate.txt:1]: You don't need `GitHub release (pre-release)` entry. Tags are mutable and you already shared the commit ID anyway.

**Review by vy — APPROVED** [review]
* I'm trusting you with `ps1` scripts, I haven't reviewed them.
* Dropped a minor remark about `MailTemplate.txt`.

File: doc/MailTemplate.txt
Comment (FreeAndNil) [reply doc/MailTemplate.txt:1]: Thanks for the hint. I removed the line.

**Pull request merged**